2020/2020-04-21.committee.md

```diff
@@ -0,0 +1,261 @@
+            - Run a mini-contest over one of the nights
+        - Could run a content session
+            - e.g. Game Theory for games with unigames
+            - e.g. "How to run a Competitive Programming Club"
+            - e.g. "How to code in PowerPoint"
+        - Nic: Could we commit to a contest and content session?
+            - Should aim to be done by the 10th of May (2020-05-10)
+            - ACTION: Nic/Ryan to organise
+    - Need to commit to a promotion plan
+        - Posters and stuff should probably be done by the 1st of May
+        (2020-05-01)
+            - Friday after next
+            - ACTION: Nic/Zahra to do the posters and stuff for Charity Vigil
+    - Nic moves that we approve any of the three charities at the
+    subcommittee's discretion
+        - Tom seconds
+        - 7:0:1, passes
+        - There will a decision by the end of tomorrow on the charity that will
+        be supported
+    - Nic moves that we approve a budget of $100 for Charity Vigil 2020
+        - Seamus seconds
+        - 8:0:0, passes unanimously
+        - Specifics of what we'll get will be discussed in the coming weeks
+        - It's likely to be a Steam key or something similar
+    - Subcommittee meets on Wednesdays at 5PM, so more information will be
+    available after that
+
+## General Business
+- CSSE feedback session
+    - Most of it was on assessments, and how they were going to work
+    - FAQ has a lot of helpful information on opting-out of Examplify
+    - You must opt-in to an ungraded pass after you know your mark
+        - You can't do it before
+    - They are looking to expand the number of units taught
+        - Are kind of strapped by all the beaurocracy
+        - There was talk of removing CITS3003 as a core unit
+    - The course is up for accreditation next year
+    - They are looking for feedback from students, with the point that the
+    changes should be minor, not major
+    - Ryan: Are the feedback sessions exclusively undergrad focussed?
+        - No, although most of the discussion was about it
+        - Ryan: Then with regards to MPE, there's not enough undergrad rigor
+        for the Software specialization
+            - You can do it without having written much code at all
+    - Ryan: There's Masters of IT: Open Source Software and Tooling unit
+        - Run by Chris MacDonald
+        - Could it perhaps become a complementary unit for the CS stream?
+- The CSSE staff are feeling a bit lonely without students in class
+    - They want feedback from students in general on how to get more out of the
+    students
+- The book
+    - Zahra: We could ask our members what they'd like information about
+
+## Previous Action Items
+- ACTION: Nic to make new channel for Charity Vigil
+    - Done
+- ACTION: Nic to pester ICON about charity vigil
+    - Done
+- ACTION: Nic/Zahra to put up a post about Code Jam 1B
+    - Done
+- ACTION: Nic/Zahra to put up a post about Kick Start
+    - lmao no
```

Comment:
Is this a 'no it is not done', or 'no it is not going to happen at all'?

Reply:
It was an "its too late".

If I remember correctly, this was partly mentioned in Zahra's report.